scripts/do-release.sh

```diff
@@ -250,6 +264,53 @@ function post_reviews_comment() {
     dryrun gh pr review "${pr_url}" --comment --body "${comment}"
 }
 
+function get_pr_body() {
+   wget "${PR_TEMPLATE}"
+   sed -ir "s/\[ \]/\[x\]/g; 0,/Is operator/d" pull_request_template.md
+   sed -r "1s/^/Release Submariner v$1\n/" pull_request_template.md
+   rm -f pull_request_template.md
+}
+
+function release_bundle() {
+
+    local bundle_version
+    bundle_version="$(echo "${release["version"]}" | cut -d'-' -f1 | cut -c2- | cut -d'.' -f1,2,3)"
+    local bundle_from_version="${release["bundle.from_version"]:-0.0.0}"
+    local bundle_channel="${release["bundle.channel"]:-$(echo alpha-"${bundle_version}" | cut -d'.' -f1,2)}"
+    local pr_body
+    pr_body=$(get_pr_body "${bundle_version}")
+
+    (
+        project=submariner-operator
+        pushd projects/${project}
+        dapper_in_dapper
+
+        make packagemanifests \
+            VERSION="${bundle_version}" \
+            FROM_VERSION="${bundle_from_version}" \
+            CHANNEL="${bundle_channel}"
+    )
+
+    if [ ! -d "${SOURCE_BUNDLE_DIR}/${bundle_version}" ]; then
+        echo "ERROR: The bundle version ${bundle_version} was not found in ${SOURCE_BUNDLE_DIR}/${bundle_version}"
+        exit 1
```

Comment:
```suggestion
        return 1
```

Better return and not exit entirely, to give callers a chance to continue/recover.